_includes/partials/callouts.html

```diff
@@ -0,0 +1,39 @@
+<div class="callout">
+  <a class="card1" href="https://bit.ly/2SbrRhe">
+    <h3>Community Calendar</h3>
+    <p class="small">Join the weekly meetings by subscribing to our community calendar</p>
+    <div class="go-corner" href="https://bit.ly/2SbrRhe">
+      <div class="go-arrow">
+        <img id="logo_meshery" src="{{ site.baseurl }}/images/logos/meshery-logo.png" alt="" />
+      </div>
```

Comment:
Considering you are already adding the icon using CSS just above this line.
I don think we need the image being added separately. The same is applicable for the code below.